dev/src/reference.ts

```diff
@@ -1349,7 +1365,38 @@ export class Query<T = DocumentData> {
   limit(limit: number): Query<T> {
     validateInteger('limit', limit);
 
-    const options = this._queryOptions.with({limit});
+    const options = this._queryOptions.with({
+      limit,
+      limitType: LimitType.First,
+    });
+    return new Query(this._firestore, options);
+  }
+
+  /**
+   * Creates and returns a new Query that only returns the last matching
```

Comment:
We should update `limit()`'s comment to mirror the wording there as well.

Reply:
Done.